### ISSUE TITLE
Source of the photo

### ISSUE THREAD

**Issue by edgartdata** [PHAROS-photoarchives/Zeri-Foundation#11]
I have created a clean copy of the data matrix: https://docs.google.com/spreadsheets/d/1QdgGkS2QjI1WewerrReNoVddCh3IK9Zi6hU11xzdNZE/edit#gid=2141014877 
and given unique IDs to the fields so Photo Provenance fields are now all PP1-.... Source of the photo is now PP3-Source of the photo. I hope it's more helpful than referencing row numbers.

For PP3 for Zeri I see the name of the source (Galleria estense) but I also see location information:
PRVS[@etichetta="Country"]
PRVP[@etichetta="District"]
PRVC[@etichetta="Town / Municipality"]
PRCM[@etichetta="Archive/ Collection"]
PRDI[@etichetta="Date IN"]
PRDU[@etichetta="Date OUT"]
PRCD[@etichetta="Repository"]

Since we were going to map this field to a people/institution authority file, may I suggest that you map the location info proper to another field?

**Comment by marilenadaquino** [comment]
Yes of course. However, these entities are not described in a dedicated authority file at the Zeri foundation, hence they do not have an ID. Plus, there may be homonyms, so I'd keep at least PRVS, PRVP, PRVC.
